blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jorge Vicente Cantero and Martin Duhem
+title: "Meet bloop, a fast tool to compile and test your project"
+---
+
+Martin and I are excited to introduce `bloop`, a new tool that we have been working on at the Scala
```

Comment:
Duhem

Reply:
Good catch, didn't think about the other Martin 😛